### DISCUSSION TITLE
GPS navigation

### DISCUSSION THREAD

**Discussion by GunTotnHippie** [meshtastic/Discussions#122] [Firmware]
I am brand new to Meshtastic.  Is it possible to add on a magnetometer/compass instead of relying on dead reckoning?    Will the firmware support it and can you swap to using a compass instead of dead reckoning in the UI?  I am looking at going with the T114 BTW.

**Comment by madeofstown** [comment]
I think youve got your definitions wonky. Dead reckoning is what you are asking for. https://en.m.wikipedia.org/wiki/Dead_reckoning

GPS is like the opposite of dead reckoning.
